src/components/StudyCreate/StudyCreateFileInput/StudyCreateFileInput.tsx

```diff
@@ -0,0 +1,23 @@
+import Button from '@mui/material/Button';
+
+interface Props {
+  message: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+function StudyCreateFileInput({ message, onChange }: Props) {
+  return (
+    <Button component='label'>
+      {message}
+      <input
+        hidden
+        type='file'
+        accept='image/jpg,impge/png,image/jpeg,image/gif'
+        name='study-image'
+        onChange={onChange}
+      ></input>
```

Comment:
닫는 태그로 수정해도 되지 않을까 싶습니다!🙄

```suggestion
      <input
        hidden
        type='file'
        accept='image/jpg,impge/png,image/jpeg,image/gif'
        name='study-image'
        onChange={onChange}
      />
```

Reply:
수정했습니다. 감사합니다 👍 447bc1eb8fcf9b960ea82057b530cbee369394e3